understanding/22/target-size-minimum.html

```diff
@@ -92,7 +100,7 @@ <h2>Intent of Target Size (Minimum)</h2>
     </section>
     <section id="benefits">
       <h2>Benefits of Target Size (Minimum)</h2>
-      <p>Having targets with sufficient target spacing can help all users who may have difficulty in confidently targeting or operating small controls. Users who benefit include, but are not limited to:</p>
+      <p>Having targets with sufficient size, or failing that, sufficient target spacing can help all users who may have difficulty in confidently targeting or operating small controls. Users who benefit include, but are not limited to:</p>
       <ul>
         <li>Users who use a mobile device where the touch screen is the primary mode of interaction;</li>
```

Comment:
might be worth adding to one of these cases that the users are using a mouse, or a head wand, or some other non-touchscreen input, to make it clear it's not just about touch.